### ISSUE TITLE
Dependency to blueimp-load-image breaks builds with Webpack

### ISSUE THREAD

**Issue by schickling** [smooch/smooch-js#307]
You recently introduced `blueimp-load-image` which seems to break builds with Webpack. See these error messages:
![screen shot 2016-07-01 at 18 21 10](https://cloud.githubusercontent.com/assets/1567498/16527699/a1d81b62-3fb8-11e6-9275-303fc2077607.png)

**Comment by schickling** [comment]
I've just seen you do the following in your `webpack.config.js`: https://github.com/smooch/smooch-js/blob/integration/make-webpack-config.js#L45-L49

However this is not something you should ask your users to need to configure...

**Comment by lemieux** [comment]
Unfortunately, `load-image` isn't really webpack friendly and that's the workaround we found (https://github.com/smooch/smooch-js#notes-about-webpack) to make it work. If you know a better way to do this, I'll be more than happy to implement it.

**Comment by schickling** [comment]
What would be the downsides of providing a webpack-precompiled version of this package?

**Comment by lemieux** [comment]
Yeah I guess we could do that. One downside would be the need to update that package whenever it changes, but it seems pretty stagnant for a while now. 

I'll add that to our things to do.

**Comment by schickling** [comment]
Any update?

**Comment by flokli** [comment]
I also ran into this issue. Would love to see smooch-js running smoothly with webpack :-)
